tests/components/modbus/test_modbus_fan.py

```diff
@@ -0,0 +1,134 @@
+"""The tests for the Modbus fan component."""
+import pytest
+
+from homeassistant.components.fan import DOMAIN as FAN_DOMAIN
+from homeassistant.components.modbus.const import (
+    CALL_TYPE_COIL,
+    CALL_TYPE_REGISTER_HOLDING,
+    CALL_TYPE_REGISTER_INPUT,
+    CONF_COILS,
+    CONF_FANS,
+    CONF_INPUT_TYPE,
+    CONF_STATE_OFF,
+    CONF_STATE_ON,
+    CONF_VERIFY_REGISTER,
+)
+from homeassistant.const import (
+    CONF_ADDRESS,
+    CONF_COMMAND_OFF,
+    CONF_COMMAND_ON,
+    CONF_NAME,
+    CONF_SLAVE,
+    STATE_OFF,
+    STATE_ON,
+)
+
+from .conftest import base_config_test, base_test
+
+
+@pytest.mark.parametrize(
+    "do_config",
+    [
+        {
+            CONF_ADDRESS: 1234,
+        },
+        {
+            CONF_ADDRESS: 1234,
```

Comment:
Duplicate ??

Reply:
Good catch! The same will be in Light too. I'll fix that.

Reply:
This same duplicate is in `switch.py`.